Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
I have a vague memory that we originally thought that [Reentrant] was need / useful to allow "read-only" data to be returned to the UI from the ChirperAccount grain's Get* methods. 
That may have been from the time when read-only methods were turned off because the ActivationTaskSchedule did not handle interleaving them correctly? 
Some where along the road we removed all the [ReadOnly] annotations on the Get* methods in this grain, so using [Reentrant] here is probably masking that intent.